backend/dropbox_client.py

```diff
@@ -6,24 +6,35 @@ def __init__(self):
             oauth2_access_token="temp_token"
         )
 
-    def upload_file_to_dropbox(self, email_contents, path, desired_file_name, access_token):
-        content_in_bytes = self.convert_email_content_to_bytes(email_contents)
-        path = "%s/%s" % (path, desired_file_name)
-
+    def upload_file_to_dropbox(self, file_contents, path, file_name, access_token):
```

Comment:
Note: Most of this code was copied over from https://github.com/rogebrd/mailman/pull/14